source/management.rst

```diff
@@ -1,3 +1,14 @@
 =============================
  The Management of the PyCQA
 =============================
+
+- Creator: Ian Cordasco
+
+- Pep8 Administrator: Ian Lee
```

Comment:
Might be worth adding 'pycodestyle' here as well for as that transition takes place